### ISSUE TITLE
Create Bridge Domain

### ISSUE THREAD

**Issue by lacheron** [carlniger/acitool#24]
Hello Carl, 
i'm sebastian, i been trying to create a bridge domain, i wasn't able to do it, 
but i could create tenant, vrf and app, but it fails to create the bridge domain

I'm running the scripts against an apic 3.1(1) version

this is the xls file that i'm using

[aci_deploy.zip](https://github.com/carlniger/acipdt/files/1939508/aci_deploy.zip)

this is the output:

![c1](https://user-images.githubusercontent.com/33548771/39145109-20c9af86-4709-11e8-873d-8e12b08993d7.PNG)

Please can you help me, thanks a lot for your time
Best Regards

**Comment by carlmontanari** [comment]
Ah I think I see what is going on. Can you change the status on your BD --> "created, modified"

This is one of the things I would like to overhaul in PDT -- I've coupled some things together that probably shouldn't have been -- for example BD/VRF binding. Running the spreadsheet as you've got this is the response I see:

{"totalCount":"1","imdata":[{"error":{"attributes":{"code":"103","text":"Cannot create Private Network (fvRsCtx); object uni\/tn-angeles\/BD-bd1\/rsctx already exists."}}}]}

Basically the BD call is pointing to the VRF (to associate it there), but the VRF already exists, because the status is "created" it will basically fail if the object already exists.

Give that a shot and let me know if that clears it up! Longer term hopefully I will get around to updating things :)

**Comment by lacheron** [comment]
Hello Carl, 

Thanks a lot for your help, i could create a bridge domain, it was just like you said
when i modified the create,modified status everything went fine. so the reason was that it called a vrf that it was created sometime before. 

thanks a lot for your help
best regards

**Comment by carlmontanari** [comment]
No problem, glad its working for you!
